.github/workflows/php-compatibility.yml

```diff
@@ -1,4 +1,4 @@
-name: PHP Compatibility
+name: PHP Linting
```

Comment:
I've left the file name the same for now so the required workflows setting in the admin doesn't need to be changed. This may or may not be a great idea.